### ISSUE TITLE
Update doc for CA1711 to mention new cases

### ISSUE THREAD

**Issue by Evangelink** [MicrosoftDocs/visualstudio-docs#5037]
With the PR https://github.com/dotnet/roslyn-analyzers/pull/3360 the rule will now also report on enums whose name ends in `Flag` or `Flags`. The rule description needs to be updated accordingly.

**Comment by Mikejo5000** [comment]
dotnet/roslyn-analyzers#3360 is approved but note merged yet. @Evangelink Do you know what analyzer version this will get shipped in?

**Comment by Evangelink** [comment]
Next one but I am not sure how things will be done. Tagging @mavasani as he will have the answer (could you proceed with the merge too please?).

**Comment by Mikejo5000** [comment]
Looks like it was merged a while ago. I'm updating the table of Suffixes | Base classes with this entry:

Flag | Flags   System.Enum

Let me know if that seems good. Thanks!

**Comment by Mikejo5000** [comment]
It raises the question whether plural forms of all the entries are caught. Maybe I just need to say "Flag"?

**Comment by Mikejo5000** [comment]
Updating. Doc should go live around noon PST. Please see my question above about plural forms in general. Thanks!

**Comment by Evangelink** [comment]
@Mikejo5000 Only `Flag` and `Flags` for enums.

Looking at the PR, I can see that we also added a new user option: https://github.com/dotnet/roslyn-analyzers/pull/3360/files#diff-1b5cdc4937a8cdfc79545945ad6c0783R697-R704

Do you want me to create a separate ticket or are you gonna handle it too?

**Comment by Mikejo5000** [comment]
I will add it. Looks like: **additional_use_results_methods**
Configure a string of allowed suffixes (separated by '|')

**Comment by Mikejo5000** [comment]
Updating. Fix should go live around 5pm PST. Let me know if you see any issues.

**Comment by Evangelink** [comment]
@Mikejo5000 There seems to be a visual glitch in the table for the newly added `Flag`/`Flags` entry.

![image](https://user-images.githubusercontent.com/11340282/88160465-f463a180-cc0e-11ea-9677-80857513c32c.png)

**Comment by Evangelink** [comment]
Also the configurability section is not following the same structure as for some other rules (e.g. [ca1303](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1303)). Shall I open a different issue?

**Comment by Mikejo5000** [comment]
Good catch. No, I'll make the fix now, and we can discuss here if there are more issues.

**Comment by Mikejo5000** [comment]
On Configurability, do you mean make the phrasing consistent? ca1303 says this instead:

"If you're running this rule from FxCop analyzers (and not with legacy analysis), this rule is configurable." 

I will update using this phrasing. Let me know if I'm missing anything here.

**Comment by Mikejo5000** [comment]
I'll merge updates. Should be live around noon.

**Comment by Evangelink** [comment]
Sorry I meant that:

1/ The sentence you are mentioning is put only once as a kind of summary.

2/ Each option is put in a headered section:
![image](https://user-images.githubusercontent.com/11340282/88202776-a61fc400-cc49-11ea-8910-d4bbbf4efd93.png)

**Comment by Mikejo5000** [comment]
Yes, that makes sense, of course! Thanks for the clarification.

**Comment by Mikejo5000** [comment]
Should be live around noon.

**Comment by Evangelink** [comment]
@Mikejo5000 Thank you! Everything looks good now :)
